jenkins/help.md

```diff
@@ -22,7 +22,7 @@ The connection configuration accepts the following parameters:
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
 |credentials|credential_username_password|None|True|Username and password|None|
-|host|string|None|True|Jenkins server URL|None|
+|host|string|None|True|Jenkins' server URL|None|
```

Comment:
```suggestion
|host|string|None|True|Jenkins server URL|None|
```

Reply:
Good catch

Reply:
Wouldn't this be like `Github server URL`?